scripts/generate-test-services.ts

```diff
@@ -137,8 +137,32 @@ if (arg === 'e2e' || arg === 'all') {
 
 if (arg === 'openapi' || arg === 'rest' || arg === 'all') {
   generateOpenApi({
-    inputDir: path.resolve('test-resources', 'openapi-service-specs'),
-    outputDir: path.resolve('test-packages', 'test-services', 'openapi'),
+    inputDir: path.resolve(
+      'test-resources',
+      'openapi-service-specs',
+      'test-service'
```

Comment:
perhaps for consistency call it `test-service-json`?

Reply:
Fair point. I thought at least in the beta development, we rely on the json version. Just like the odata test entity, `test-entity`/`test-service` are the default names that we want to maintain.